manager/integration/tests/test_csi_snapshotter.py

```diff
@@ -1188,3 +1188,304 @@ def finalizer():
                               'default')
 
     request.addfinalizer(finalizer)
+
+
+def test_csi_volumesnapshot_backing_image_basic():
+    """
+    Test Create/Delete BackingImage using VolumeSnapshot with a given Volume
+
+    Setup
+    - Create a VolumeSnapshotClass with type `bi`
+        ```
+        kind: VolumeSnapshotClass
+        apiVersion: snapshot.storage.k8s.io/v1
+        metadata:
+            name: longhorn-snapshot-vsc
+        driver: driver.longhorn.io
+        deletionPolicy: Delete
+        parameters:
+            type: bi
```

Comment:
Do we want to change the `type: bi` to `type: backing-image`? :)

Reply:
this follows the naming style of other two types which are `snap` for "snapshot" and `bak` for "backup"
This type will be used as part of the snapshotHandle as ID in the form of `bi://XXXX/XXX`
so better keep it short.
Thanks!